empiric-package/empiric/core/config.py

```diff
@@ -0,0 +1,93 @@
+import os
+
+from empiric.core.types import INTEGRATION, MAINNET, STAGING, TESTNET, Network
+from starknet_py.net.models import StarknetChainId
+from typing_extensions import Protocol
+
+
+# TODO (rlkelly): We need a consistent way to implement interfaces.  This seems to be the preferred method
+class IConfig(Protocol):
+    ADMIN_ADDRESS: int
+    DEFAULT_AGGREGATION_MODE: int
+    NETWORK: Network
+    CHAIN_ID: int
+    PUBLISHER_REGISTRY_ADDRESS: int
+    ORACLE_CONTROLLER_ADDRESS: int
+    GATEWAY_URL: str
+
+
+class BaseConfig(IConfig):
+    ADMIN_ADDRESS = 0x0704CC0F2749637A0345D108AC9CD597BB33CCF7F477978D52E236830812CD98
+    DEFAULT_AGGREGATION_MODE = 0
+    # this indicates that the integration network uses goerli
+    CHAIN_ID = StarknetChainId.TESTNET
+
+
+class TestnetConfig(BaseConfig):
+    NETWORK = TESTNET
+    GATEWAY_URL = "https://alpha4.starknet.io"
+    ORACLE_CONTROLLER_ADDRESS = (
+        0x012FADD18EC1A23A160CC46981400160FBF4A7A5EED156C4669E39807265BCD4
+    )
+    PUBLISHER_REGISTRY_ADDRESS = (
+        0x0743E8140A56D5EE9ED08EB77A92BCBCF8257DA34AB2A2EE93110709E61AB11A
+    )
+
+
+class IntegrationNetworkConfig(BaseConfig):
+    NETWORK = INTEGRATION
```

Comment:
Change this to `NETWORK = "https://external.integration.starknet.io"`

Reason: The integration net is not recognized by starknet.py and needs to be used as a custom gateway. This means the network needs to equal the gateway_url in this case